F5-NGINX-team-notes.md

```diff
@@ -1,12 +1,10 @@
 # For F5/NGINX Employees
 
-This repository is a functional mirror. We want to do as much of our work as possible in the
-public repository. However, if you are working with:
+This repository is public, and we do as much of our work as possible in the public as a commitment to open source.
 
-- Security content, including personally identifying information (PII).
-- Content / features that are not yet ready to be announced.
+Content which cannot be publicised before release is handled slightly differently: please read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.
```

Comment:
```suggestion
If you're working with content which cannot be publicized before release is handled slightly differently: please read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.
```

Reply:
I need to re-read this in context but I think I specifically avoided the phrasing of "If you're working" due to repetition.

Reply:
I think "active voice" is important here